ads-with-uniregistry.rst

```diff
@@ -0,0 +1,73 @@
+.. post:: 
+   :tags: advertising, sustainability, draft
+
+Uniregistry sponsors Read the Docs and Open Source
+==================================================
+
+Today we're excited to announce an important sponsorship partner in our `Ethical Advertising`_ campaign: `Uniregistry`_.  Our goal with our ethical advertising program is to provide important funding for open source, and show that it can be done ethically.
+
+Domain registration was identified early as a natural partner to our program, because it sits in the stack of necessary infrastructure for all of us that work on making the internet. We wanted the right partner, because historically we feel that domain registration companies left room for improvement in the customer experience. This is post covers a few of the criteria we used to reach the conclusion to partner with Uniregistry.
```

Comment:
A few things on this:

* Lead is good!
* "We wanted the right partner, because ... other registrars suck at UX" is a weak justification for this partnership. I think the point we are trying to make is that we wanted a partner that was relevant to our users and believed in what we were doing. Uniregistry happens to provide a great user experience on top of this.
* "left room for improvement in the customer experience" can be condensed to something easier to understand
* I avoid explicit headings like "Conclusion" and explicit content like "This is post covers ...". Language like this is unnatural outside dissertations.